set_env.c

```diff
@@ -0,0 +1,63 @@
+#include <stdlib.h>
+#include <unistd.h>
+#include <string.h>
+
+extern char **environ;
```

Comment:
This is now already included in `hsh.h`
```suggestion
#include <string.h>
#include "hsh.h"
```